### ISSUE TITLE
Question about norm2d

### ISSUE THREAD

**Issue by huangwei9** [shachoi/HANet#8]
Have you ever tried to experiment on a single GPU?
In your code:
```
self.attention_first = nn.Sequential(
               nn.Conv1d(in_channels=in_channel, out_channels=mid_1_channel,
               kernel_size=1, stride=1, padding=0, bias=False),
               Norm2d(mid_1_channel),
               nn.ReLU(inplace=True))  
 ```
By default, norm2d is batchnorm2d, and batchnorm2d expected 4D input but it seems got 3D input.

**Comment by sunghac** [comment]
Yes, you are right. Norm2d, the naming of the code causes confusion.
However, by default, Norm2d denotes "[torch.nn.SyncBatchNorm](https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html#torch.nn.SyncBatchNorm)". SyncBatchNorm supports N-dimensional input.
If you want to use typical BatchNorm, you should change Norm2d into nn.BatchNorm1d(mid_1_channel).
Thanks!

**Comment by huangwei9** [comment]
> Yes, you are right. Norm2d, the naming of the code causes confusion.
> However, by default, Norm2d denotes "[torch.nn.SyncBatchNorm](https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html#torch.nn.SyncBatchNorm)". SyncBatchNorm supports N-dimensional input.
> If you want to use typical BatchNorm, you should change Norm2d into nn.BatchNorm1d(mid_1_channel).
> Thanks!


Thank you for your reply!
